libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -44,6 +44,9 @@ singleton AP_AHRS method get_EAS2TAS float
 singleton AP_AHRS method get_variances boolean float'Null float'Null float'Null Vector3f'Null float'Null
 singleton AP_AHRS method set_posvelyaw_source_set void uint8_t 0 2
 singleton AP_AHRS method get_vel_innovations_and_variances_for_source boolean uint8_t 3 6 Vector3f'Null Vector3f'Null
+singleton AP_AHRS method set_home boolean Location
```

Comment:
This races with the vehicle code, because `set_home` doesn't have any semaphore protection (AP_AHRS doesn't use the semaphore around this type of thing).